src/index/CompressedRelation.cpp

```diff
@@ -388,21 +388,10 @@ CompressedRelationReader::IdTableGenerator CompressedRelationReader::lazyScan(
       });
 }
 
-// Helper function that enables a comparison of a triple with an `Id` in
-// the function `getBlocksForJoin` below.
-//
-// If the given triple matches `col0Id` of the given `ScanSpecification`, then
-// `col1Id` is returned.
-// respective other `Id` of the triple is returned.
```

Comment:
I dropped this line from the documentation when moving it into the header because it seemed incomplete. This needs fixing.